boards/arm/mercury_xu/mercury_xu_board.yaml

```
@@ -0,0 +1,18 @@
+# Copyright (c) 2019, Antmicro
+identifier: mercury_xu_board
+name: MERCURY-XU-BOARD
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gccarmemb
```

`gccarmemb` is obsolete.

```suggestion
  - gnuarmemb
```